CONTRIBUTING.md

```diff
@@ -57,21 +56,45 @@ Herein lies the contribution guidelines for helping out with this project. Do ta
 
 * There should be a single line break between tasks
 * Every task (except prelim tasks) should have, at a minimum and when applicable, the following tags:
-   * Severity level (cat1, cat2, high, etc)
-   * Identification number (example, the vulnerabilty ID number in the case of RHEL6 STIG)
-   * audit/patch
-* Tags can be either bracketed or multi-line
-* Tasks should happen sequentially as they appear in the given standard. 
+    * Severity level (cat1, high, cat2, medium, etc.)
+    * Identification number (example, the vulnerability ID number in the case of RHEL6 STIG)
+    * audit/patch
+* Tags can be either bracketed or multi-line, though multi-line is preferred. If multi-line tags are used, they should be indented four spaces just like module arguments above.
+```yml
+# This
+- name: Create a directory
+  file:
+      state: directory
+      path: /tmp/deletethis
+  tags:
+      - files
+      - debug
+
+# Not This
+- name: Create a directory
+  file:
+    state: directory
+    path: /tmp/deletethis
+  tags:
+    - files
+    - debug
+```
+* Tasks should run sequentially by vulnerability ID as listed in the given standard. 
 * Every task must be named and should adhere to the following convention:
 ```yml
-- name: "| $severity | $id_number | patch/audit |\n
-        $description_provided_by_standard"
+- name: "$severity | $id_number | PATCH/AUDIT | $description_provided_by_standard"
 
-- name: "| HIGH | V-38476 | PATCH |\n
-        Vendor-provided cryptographic certificates must be installed to verify the integrity of system software."
+- name: "HIGH | V-38476 | PATCH | Vendor-provided cryptographic certificates must be installed to verify the integrity of system software."
+```
+* Every standard implemented must consist of at least two sequential tasks: one that conducts a check and registers the results to a variable, and another that applies the standard. Note that the task which applies the standard does not necessarily have to use the registered variable from the prior task.
+* There should only be one standard remediated or checked per task, even if several remediations could be combined into a single task. The goal of this role is granular remediation.
+* If multiple standards must be combined into a single task, the name should adhere to the following convention:
```

Comment:
The only scenario I'm imagining is a template task that copies a template that remediates multiple items. I believe `audit.rules` is a good example, but I haven't gotten into refactoring any of these tasks yet.

Hypothetically, imagine this template file:

```jinja
...
parameter1 = foo
parameter2 = bar
{% if rhel6stig_cat1 %}
some settings that fix a CAT I issues
{% endif %}
{% if rhel6stig_cat2 %}
some settings that fix CAT II issues
{% endif %}
...
```

I'm not sure of a way to break that into multiple tasks without descending into `lineinfile` hell.